src/actions/posts.ts

```diff
@@ -482,7 +482,11 @@ export function pinPost(postId: string) {
                 ...post,
                 is_pinned: true,
                 update_at: Date.now(),
-            }));
+            }),
+            {
```

Comment:
Would you mind fixing the indentation here so that the arguments are indented relative to the function call? I'm going to make a note to look at if ESLint can enforce that for us since I know we're not always consistent with it
```javascript
actions.push(
    receivedPost({
        // ...
    }),
    {
        type: ChannelTypes.INCREMENT_PINNED_POST_COUNT,
        id: post.channel_id,
    }
);
```